### ISSUE TITLE
After using Docker, the logs terminates with only three lines

### ISSUE THREAD

**Issue by ydmykr** [microsoft/vscode-remote-release#7422]
Version: 1.72.2 (user setup)
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-26
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22000
Sandboxed: No

VSCode Version: 1.72.2
Local OS Version: Windows 11 Pro
Remote OS Version: Docker Engine 20.10.18 (Installed using Chocolatey. Not using Docker Desktop)
Remote Extension/Connection Type: SSH/Containers/WSL/Server
Logs:

```
[152129 ms] Dev Containers 0.255.4 in VS Code 1.72.2 (d045a5eda657f4d7b676dedbfa7aab8207f8a075).
[152129 ms] Start: Run: ssh raspberrypi /bin/sh
[152140 ms] Start: Run in host: id -un
```

After using Docker, the logs terminates with only three lines.

It did not operate correctly. Is there any problem? Thank you for your response.

**Comment by chrmarti** [comment]
When you run `ssh raspberrypi` in the terminal, does it require any keyboard input before connecting (e.g., for a password or passphrase?

**Comment by ydmykr** [comment]
When I launch the ssh session I put my password in.

**Comment by chrmarti** [comment]
This looks like https://github.com/microsoft/vscode-remote-release/issues/7355 and should be fixed in Dev Containers 0.262.1-pre-release. Could you check if that version fixes it for you? Thanks.

**Comment by ydmykr** [comment]
Hey, thanks for the content, it solves the problem
